File: rudder-sdk-core/src/main/java/com/rudderlabs/android/sdk/core/RudderOSInfo.java
package com.rudderlabs.android.sdk.core;

import android.os.Build;
import com.google.gson.annotations.SerializedName;
import com.rudderlabs.android.sdk.core.BuildConfig;

class RudderOSInfo {
    @SerializedName("name")
    private String name = "Android";
    @SerializedName("version")
    private String version = Build.VERSION.RELEASE;
}

File: models/src/main/java/com/rudderstack/android/models/android/RudderOSInfo.kt
package com.rudderstack.android.models.android

import com.google.gson.annotations.SerializedName

internal class RudderOSInfo (
    @SerializedName("name")
    private val name : String = "Android",

    @SerializedName("version") // = Build.VERSION.RELEASE
    private val version: String? = null
)